public/app/core/components/NavBar/NavBarItem.tsx

```diff
@@ -20,6 +22,23 @@ export interface Props {
   link: NavModelItem;
 }
 
+// Maps the ID of the nav item to a translated phrase to later pass to <Trans />
+// Because the navigation content is dynamic (defined in the backend), we can not use
+// the normal inline message definition method.
+// Keys MUST match the ID of the navigation item, defined in the backend.
+// see pkg/api/index.go
+const TRANSLATED_MENU_ITEMS: Record<string, MessageDescriptor> = {
+  home: defineMessage({ id: 'nav.home', message: 'Home' }),
+  dashboards: defineMessage({ id: 'nav.dashboards', message: 'Dashboards' }),
+  'manage-dashboards': defineMessage({ id: 'nav.manage-dashboards', message: 'Browse' }),
+  playlists: defineMessage({ id: 'nav.playlists', message: 'Playlists' }),
+  snapshots: defineMessage({ id: 'nav.snapshots', message: 'Snapshots' }),
+  'library-panels': defineMessage({ id: 'nav.library-panels', message: 'Library panels' }),
+  'new-dashboard': defineMessage({ id: 'nav.new-dashboard', message: 'New dashboard' }),
+  folder: defineMessage({ id: 'nav.folder', message: 'New folder' }),
+  import: defineMessage({ id: 'nav.import', message: 'Import' }),
+};
```

Comment:
doesn't have to block this PR, but it might be nice if we can think about a way to have a contract test here. from past experience, relying on us as developers to ensure the id is the same in 2 places will inevitably fail at some point.

Reply:
also we have a comment at this end but not in the api end. which means people will very quickly add things to the backend api call that won't be translated in the frontend.